src/pages/workspace/accounting/qbd/export/QuickbooksDesktopOutOfPocketExpenseEntitySelectPage.tsx

```diff
@@ -64,22 +74,23 @@ function QuickbooksDesktopOutOfPocketExpenseEntitySelectPage({policy}: WithPolic
 
     const selectExportEntity = useCallback(
         (row: MenuItem) => {
-            if (row.value !== reimbursable) {
-                QuickbooksDesktop.updateQuickbooksDesktopExpensesExportDestination(
+            const account = row?.accounts?.at(0)?.id;
+            if (row.value !== reimbursable && policyID && account) {
```

Comment:
Can you explain these changes? When I tested with my account, the `account` is undefined,  so it does nothing when you change export out-of-pocket

https://github.com/user-attachments/assets/72753830-f574-4da7-84ba-00f0c4f247f4